### ISSUE TITLE
CPU is running very slow + GPU-process consuming lots of memory (VSCode 1.35.0)

### ISSUE THREAD

**Issue by AasmundEndresen** [microsoft/vscode#75062]
- VSCode Version: 1.35.0
- OS Version: Windows 10, version 1809

I'm working in jQuery/js/php/css - not running any extensions.

After working for approx. 5-10 minutes the CPU usage explodes whenever I type.  It's quickly lagging behind 20+ characters even at a slow/moderate pace of writing code.

Opening the process explorer on a second monitor as I type reveals the `window ([filename] - [directory] - Visual Studio Code)` process to be the culprit.  It quickly runs up to 50%+ CPU usage only after writing a few characters.  Also the `gpu-process` is showing abnormally high memory usage (500MB+) even when I'm not doing anything.

As mentioned above there is no extensions to disable.  Any other programs are running just fine.  Windows task manager is also showing that particular processl to have an extremely high memory and cpu usage even when inactive, and a high `power usage trend`.

The repo can't be shared as it's private, and I'm under a NDA with my employer - the issue is urgent as we're aiming to have my current work go live in production Monday/Tuesday - and if we can't do it by then we have to push back an entire week.

I've been very pleased with VScode in the past, but as of lately there have been many bugs and memory leaks such as the TS/styled-components issue that have been taking months to fix and impacting performance by alot whilst adding new features appears to be prioritised.

Does this issue occur when all extensions are disabled?: Yes

**Comment by WSLUser** [comment]
First step to try is upgrade to Insiders and see if the issue still exists. If it does, and you can provide a test repro, that would allow for a faster response and potential fix.

**Comment by egamma** [comment]
Also to narrow down a performance issue please follow the instructions in this document https://github.com/microsoft/vscode/wiki/Performance-Issues

**Comment by futagoza** [comment]
I also seem to be having the issue more frequently now. It used to happen like every other week, but after updating VS Code to 1.35.0 it seems to happen like once every half an hour.

- VS Code 1.35.0
- Windows 10 x64, 1809
- AMD A10 CPU, 8GB Ram

This seems to be an issue when using JavaScript/Typescript files only, as I've only noticed it when I'm working on a `.js` or `.ts` file. When it occurs usually the type system starts to get slow and unresponsive first, leading to a `Loading...` hover message, with increased CPU and memory usage.

Beore I used to restart the editor, but after this started regurly, I've found that there's a simpler solutuin: this will only continue until I restart the JavaScript/TypeScript extension, at which point the CPU and memory usage return to normal, and the type system is responding instantly again.
